src/pages/workspace/categories/WorkspaceCategoriesPage.tsx

```diff
@@ -319,9 +367,10 @@ function WorkspaceCategoriesPage({route}: WorkspaceCategoriesPageProps) {
             );
         }
 
+        const policyHasAccountingConnections = hasAccountingConnections(policy);
```

Comment:
Can we move this outside the method at the top so that we can also use it in the import spreadsheet button conditions?